### ISSUE TITLE
Reorganize the file structure of the server

### ISSUE THREAD

**Issue by Mariocj89** [etcaterva/EtCaterva-Ansible#59]
Some logs are wihin the app folder, others in /data, come with an more "standard" approach

**Comment by palvarez89** [comment]
I don't think that migrating mongodb data to `/var/lib/mongodb` should happen automatically, although we could in Ansible check if the old location has the data, shutdown mongo, move the data, and restart mongo...

But again, I feel like doing the migration manually.

**Comment by Mariocj89** [comment]
I was planing to do it manually :)

**Comment by palvarez89** [comment]
@mariocj89 any new opinions on this? It's the perfect time to rethink thisnof wanted. For now I've been putting things in the default places.
